### PR TITLE
fix: filter never achieved streaks

### PR DESCRIPTION
**Pull request by dhenkel92**
This PR prevents something like this

image

Streaks have to be generated in advance for evaluation. Unfotunately, this doesn't automatically mean that we'll reach a first record.
Therefore, we hide streaks that never had a record yet.

### CODE REVIEW

File: scripts/test/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const moment = require('moment');
+
+const date = moment().add('5 minutes').toDate();
+console.log(date);
+
+const date2 = moment().add('5', 'm').toDate();
+console.log(date2);
```
Comment (Jonasdoubleyou) [inline scripts/test/index.js:7]: ?

File: scripts/test/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (Jonasdoubleyou) [inline scripts/test/package.json:1]: ?

**Review by Jonasdoubleyou — APPROVED** [review]

File: scripts/test/index.js (hunk 1 above)
Comment (dhenkel92) [reply scripts/test/index.js]: This is why you have reivews....
I've comitted a script that I was using for testing.

**Review by Jonasdoubleyou — APPROVED** [review]

File: scripts/test/index.js (hunk 1 above)
Comment (Jonasdoubleyou) [reply scripts/test/index.js]: nah, thats why you self-review before requesting ;)

Comment (dhenkel92) [reply scripts/test/index.js]: Not if you are in a hurry and don't do it properly 😄 

Therefore we have a second layer of security.

**Pull request merged**